drivers/adc/Kconfig.ad405x

```diff
@@ -11,6 +11,13 @@ config ADC_AD405X
 	help
 	  Enable ADC driver for ADI AD405X.
 
+config AD405X_STREAM
+	bool "Use FIFO to stream data"
+	select AD405X_TRIGGER
+	depends on SPI_RTIO
```

Comment:
`depends on ADC_AD405X` I had to fix the below bleeding Kconfig the other day, let's not add more